### PR TITLE
Add concentration keyword

### LINKED ISSUE #8: Concentration
Concentration should be a keyword. No more concentrating "as if on a spell," which is verbose at best and confusing at worst (in the case of Rage and a non-spell concentration effect, for instance).

### PR DESCRIPTION
**Pull request by bwhetherington**
This implements a description of a Concentration keyword as discussed in #8 

The wording was based on the PHB description of concentration [here](https://5e.tools/conditionsdiseases.html#concentration_phb), but with wording changed to be more generic.

### CODE REVIEW

File: src/SUMMARY.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # Summary
 
-[Introduction](README.md)
+[Introduction](./README.md)
```
Comment (JamesMcClung) [inline src/SUMMARY.md:3]: Sneaky bugfix

File: src/SUMMARY.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
 
 # Rules
 
-- [Combat](rules/combat.md)
+- [Combat](./rules/combat.md)
+- [Keywords](./rules/keywords.md)
```
Comment (JamesMcClung) [inline src/SUMMARY.md:8]: We gotta add newlines to the end of each file to avoid diffs like this

File: src/rules/keywords.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+Some spells and other effects require you to maintain concentration in order to keep them active. If you lose concentration, such an effect ends.
+
+If a spell must be maintained with concentration, that fact appears in its Duration entry, and the spell specifies how long you can concentrate on it. You can end concentration on any such effect at any time (no action required).
+
+Normal activity, such as moving and attacking, doesn't interfere with concentration. The following factors can break concentration:
+
+- Invoking another effect that requires concentration. You lose concentration if you invoke another effect that requires concentration. You can't concentrate on two effects at once.
+- Taking damage. Whenever you take damage while you are concentrating, you must make a Constitution saving throw to maintain your concentration. The DC equals 10 or half the damage you take, whichever number is higher. If you take damage from multiple sources, such as an arrow and a dragon's breath, you make a separate saving throw for each source of damage.
+- Being incapacitated or killed. You lose concentration on an effect if you are incapacitated or if you die.
```
Comment (JamesMcClung) [inline src/rules/keywords.md]: Are the first sentences here supposed to be in bold?

File: src/rules/keywords.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+- The DM might also decide that certain environmental phenomena, such as a wave crashing over you while you're on a storm-tossed ship, require you to succeed on a DC 10 Constitution saving throw to maintain concentration.
```
Comment (JamesMcClung) [inline src/rules/keywords.md]: Should this be its own paragraph?

File: src/rules/keywords.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
+# Keywords
+
+## Concentration
+
+Some spells and other effects require you to maintain concentration in order to keep them active. If you lose concentration, such an effect ends.
+
+If a spell must be maintained with concentration, that fact appears in its Duration entry, and the spell specifies how long you can concentrate on it. You can end concentration on any such effect at any time (no action required).
```
Comment (JamesMcClung) [inline src/rules/keywords.md]: The last sentence should probably go somewhere else, since it's not like you can only drop conc on spells

**Comment by JamesMcClung** [comment]
Looks good to me, aside from the nitpicks above

File: src/rules/keywords.md (hunk 4 above)
Comment (bwhetherington) [reply src/rules/keywords.md]: Yep.

File: src/rules/keywords.md (hunk 3 above)
Comment (bwhetherington) [reply src/rules/keywords.md]: They are indeed.

**Review by JamesMcClung — APPROVED** [review]

**Pull request merged**